### PR TITLE
Add Codeready Workspaces feature files.

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: Radim Hopp <rhopp@redhat.com>

### CODE REVIEW

**Comment by rhopp** [comment]
@makambalaji  PTAL ;-)

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    Background:
+        Given Red Hat CodeReady Workspaces operator is installed on the cluster
+        And user has logged in as a basic user
+        And user is at developer perspective
+
+    @smoke
+    Scenario: CRW is shown in Applications menu in Masthead
+        When user click on Application button in Masthead
+        Then Codeready Workspace entry is present
```
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature:12]: Logged in as basic user 
Unable to view application launcher itself
image
Do we need any other actions to perform before this scenario?

**Review by makambalaji — CHANGES_REQUESTED** [review]
Unable to execute this scenarios on the cluster, Please add if any other setup is required

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhopp) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature:12]: @makambalaji Yes. It's defined in "Background section" - `Given Red Hat CodeReady Workspaces operator is installed on the cluster`
If you want to try it yourself, you need to install CRW operator from OperatorHub and then create CheCluster CR (default is OK). Then wait fo CRW instance to be properly installed (can take few minutes).

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Feature: Basic Codeready Workspaces usage
```
[the remark this answers is not in the document]
Comment (rhopp) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature:1]: Will do.

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature:12]: yeah, I installed and logged in with 2 different users

**Review by makambalaji — CHANGES_REQUESTED** [review]
Please add @crw tag, Remaining everything looks good to me

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (crw) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/crw-basic.feature:2]: To be clear, that's my github user name you're mentioning.

File: frontend/packages/dev-console/integration-tests/features/codeready-workspaces/open-app-in-crw.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Feature: Opening Codeready Workspaces from Topology view for existing app
```
[the remark this answers is not in the document]
Comment (crw) [reply frontend/packages/dev-console/integration-tests/features/codeready-workspaces/open-app-in-crw.feature:2]: You know you can use backticks like `@crw` right?

**Pull request merged**